src/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -158,7 +143,8 @@
     <EmbeddedResource Include="Resources\Misc\Signed.exe" />
   </ItemGroup>
   <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+    <ProjectReference Include="..\src\System.Reflection.Metadata.csproj" />
```

Comment:
Why do we need these? are these two not in the targeting pack because they don't have a reference assembly? If so, I believe that @ericstj has merged his stuff so you shouldn't need this any longer.